packages/bot-web-ui/src/components/dashboard/dashboard.tsx

```diff
@@ -96,6 +99,8 @@ const Dashboard = ({
         }
     };
 
+    const tour_ended = React.useRef(true);
```

Comment:
@farabi-deriv Additionally I suggest you follow our guidelines ([the link is here](https://github.com/binary-com/deriv-app/blob/master/docs/JavaScript/README.md))
![Screenshot 2022-11-22 at 17 44 06](https://user-images.githubusercontent.com/103181650/203343354-05ae790d-1005-4a09-983d-a5b4ee6fd1e2.png)
Better name it `is_ tour_ended`or somehow similar

Reply:
sure. done ✅